sample/authorized.yaml

```diff
@@ -0,0 +1,61 @@
+setup:
+  - type: api
+    uri: /graphql
+    method: POST
```

Comment:
I believe you don't need this file, do you?

Reply:
Yup, thanks.